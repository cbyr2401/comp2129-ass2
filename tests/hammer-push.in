PUSH a
PUSH b
PUSH c
PUSH d
PUSH e
PUSH f
SET a 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30
LIST KEYS
LIST ENTRIES
PUSH a 0 -1 -2 -3 -4 -5 -6
LIST KEYS
LIST ENTRIES
SET catalan 14 42 132 429 1430 4862
LIST ENTRIES
PUSH catalan 5 2 1
LIST KEYS
LIST ENTRIES
PUSH b 1 2 3 4 5
SET b 1
LIST KEYS
LIST ENTRIES
PUSH b 1 2 3 4 5
LIST ENTRIES
BYE