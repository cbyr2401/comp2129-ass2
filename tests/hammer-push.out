> no such key

> no such key

> no such key

> no such key

> no such key

> no such key

> ok

> a

> a [1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30]

> ok

> a

> a [-6 -5 -4 -3 -2 -1 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30]

> ok

> catalan [14 42 132 429 1430 4862]
a [-6 -5 -4 -3 -2 -1 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30]

> ok

> catalan
a

> catalan [1 2 5 14 42 132 429 1430 4862]
a [-6 -5 -4 -3 -2 -1 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30]

> no such key

> ok

> b
catalan
a

> b [1]
catalan [1 2 5 14 42 132 429 1430 4862]
a [-6 -5 -4 -3 -2 -1 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30]

> ok

> b [5 4 3 2 1 1]
catalan [1 2 5 14 42 132 429 1430 4862]
a [-6 -5 -4 -3 -2 -1 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30]

> bye
